### PR TITLE
DIV-3660-New fields

### PR DESCRIPTION
**Pull request by mounikahmcts**
# Description

D8SeparationTimeTogetherPermitted, D8LivedTogetherMoreTimeThanPermitted are added as part of new separation dates change

Fixes # DIV-3660

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/CCDCaseToDivorceMapper.java [hunk 1]
```diff
@@ -96,6 +96,8 @@
         target = "reasonForDivorceDecisionDate")
     @Mapping(source = "reasonForDivorceLivingApartDate", dateFormat = SIMPLE_DATE_FORMAT,
         target = "reasonForDivorceLivingApartDate")
+    @Mapping(source = "separationTimeTogetherPermitted", target = "separationTimeTogetherPermitted")
+    @Mapping(source = "livedTogetherMoreTimeThanPermitted", target = "livedTogetherMoreTimeThanPermitted")
```
Comment (qzhou-hmcts) [inline src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/CCDCaseToDivorceMapper.java]: Mapping annotation is not needed if you do manual mapping

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java [hunk 2]
```diff
@@ -110,6 +110,8 @@
         target = "reasonForDivorceDecisionDate")
     @Mapping(source = "reasonForDivorceLivingApartDate", dateFormat = SIMPLE_DATE_FORMAT,
         target = "reasonForDivorceLivingApartDate")
+    @Mapping(source = "separationTimeTogetherPermitted", target = "separationTimeTogetherPermitted")
+    @Mapping(source = "livedTogetherMoreTimeThanPermitted", target = "livedTogetherMoreTimeThanPermitted")
```
Comment (qzhou-hmcts) [inline src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java]: same here, Mapping annotation is not needed if you do manual mapping

Comment (mounikahmcts) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/DivorceCaseToCCDMapper.java]: > same here, Mapping annotation is not needed if you do manual mapping

Removed

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/CCDCaseToDivorceMapper.java (hunk 1 above)
Comment (mounikahmcts) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/mapper/CCDCaseToDivorceMapper.java]: > Mapping annotation is not needed if you do manual mapping

Removed

**Pull request merged**